06.bowling_object/game.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+require './frame'
+
+class Game
+  def initialize(argv = ARGV[0])
+    @marks = argv.split(',')
+  end
+
+  def score
+    frames = build_frames
+    calculate_normal_score(frames) + calculate_bonus_score(frames)
```

Comment:
```suggestion
    frames.take(10).sum do |frame|
      frame.calc_score(frames)
    end
```

みたいにして、「ボーナス点の計算も含めて、各フレームの得点計算はFrameクラスが責務を持つ」としてしまってもいいかも。

自分が何番目のフレームかはFrame.newするタイミングで教えてあげるイメージです。

Reply:
承知しました！修正します！